support/cas-server-support-ehcache-ticket-registry/src/test/java/org/apereo/cas/ticket/registry/EhCacheActuatorTests.java

```diff
@@ -0,0 +1,63 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.config.CasCoreHttpConfiguration;
+import org.apereo.cas.config.CasCoreServicesConfiguration;
+import org.apereo.cas.config.CasCoreTicketCatalogConfiguration;
+import org.apereo.cas.config.CasCoreTicketIdGeneratorsConfiguration;
+import org.apereo.cas.config.CasCoreTicketsConfiguration;
+import org.apereo.cas.config.CasCoreUtilConfiguration;
+import org.apereo.cas.config.EhcacheTicketRegistryConfiguration;
+import org.apereo.cas.config.EhcacheTicketRegistryTicketCatalogConfiguration;
+
+import net.sf.ehcache.CacheManager;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.mail.MailSenderAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cache.ehcache.EhCacheCacheManager;
+import org.springframework.cloud.autoconfigure.RefreshAutoConfiguration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Ensure cache names array initialized in Spring CacheManager.
+ *
+ * @author Hal Deadman
+ * @since 6.1.0
+ */
+@SpringBootTest(classes = {
+    RefreshAutoConfiguration.class,
+    EhcacheTicketRegistryConfiguration.class,
+    EhcacheTicketRegistryTicketCatalogConfiguration.class,
+    CasCoreTicketCatalogConfiguration.class,
+    CasCoreTicketsConfiguration.class,
+    CasCoreTicketIdGeneratorsConfiguration.class,
+    CasCoreServicesConfiguration.class,
+    CasCoreHttpConfiguration.class,
+    CasCoreUtilConfiguration.class,
+    MailSenderAutoConfiguration.class
+}, properties = {
+    "cas.ticket.registry.ehcache.maxElementsOnDisk=100",
+    "cas.ticket.registry.ehcache.maxElementsInMemory=100",
+    "cas.ticket.registry.ehcache.shared=true",
+    "spring.mail.host=localhost",
+    "spring.mail.port=25000",
+    "spring.mail.testConnection=false"
+})
+public class EhCacheActuatorTests {
```

Comment:
This doesn't seem right; The class is not actually testing the actuator endpoint and that's what you should be doing, or you may rename the class to reflect it true nature. Since the problem was witnessed with the actuator endpoint, that would be best to test. You want to enable the endpoint, inject it and then call the right method to see what it returns in tandem with the ehcache changes above.

Reply:
I updated this to use the CachesEndpoint but I just created it in the test rather than injecting it, which is what they do in the spring boot tests for the endpoint. Do you think there is value in injecting it? I am just making sure that the CacheManager used by the endpoint knows about the ticket registry caches and leaving the rest of the CachesEndpoint testing to spring boot.

Reply:
Sounds good. Looks fine to me. Thanks!